src/doc/rustc/src/codegen-options/index.md

```diff
@@ -541,6 +541,21 @@ Supported values for this option are:
 - `symbols` - same as `debuginfo`, but the rest of the symbol table section is
   stripped as well if the linker supports it.
 
+## symbol-mangling-version
+
+This option controls the [name mangling] format for encoding Rust item names
+for the purpose of generating object code and linking.
+
+Supported values for this option are:
+
+* `v0` — The "v0" mangling scheme. The specific format is not specified at
+  this time.
```

Comment:
I think the RFC(?) is intended to be a relatively stable description of the format? We (presumably) may update/alter it a little, but in general it's my impression that the expectation is that v0 will remain relatively stable, with future major iterations going in under v1, v2, etc (if they occur), or potentially alternative naming schemes like "short" or "hash" for short 64-bit identifiers, for example.

Reply:
This was just my impression from the discussion at https://github.com/rust-lang/rust/pull/90128#issuecomment-948569123, but perhaps I misread that. I also saw that there were still discussions about the format of things like `.llvm` suffixes, and #89917 seemed that there was still ongoing discussion.  

If you want, I could change that to link to RFC 2603.  I would prefer to move that documentation into the rustc book, since I don't consider RFCs to be living documents, and changing things in the RFC repo can be a pain. But I imagine that would be a more difficult change (getting people to agree with that).

Reply:
@ehuss any updates on this?

Reply:
Per that link at https://github.com/rust-lang/rust/pull/90128#issuecomment-948569123, my understanding was that `-C symbol-mangling-version=v0` was explicitly stabilized without specifying what the `v0` format means.  Per my comment above, it seems like there are still some specifics of the format still under discussion.

I'm just waiting for feedback from Mark. I might suggest some options:

* Merge as-is.  Update this text once the `v0` format is finalized.
* Link to RFC 2603.  I'd prefer to not do that, since that implies that is an official description that is finalized.
* Migrate the text of RFC 2603 into the rustc book.  This is what I'd like to see long-term, but I think whatever outstanding concerns should be resolved first.

Reply:
I think we should proceed as is: a separate PR importing the RFC and finalizing any details makes sense to me.

I do think that our intent should be to make it a specified format we don't really change, in part because in practice it is a hassle to change it in ways that break demangling (in gdb, perf, etc).

So sounds like we're in agreement on the trajectory and next step; resolving this thread. I missed this comment earlier, sorry.